diff --git a/rich/console.py b/rich/console.py
index 3ec9a8aa..8e5c7b19 100644
--- a/rich/console.py
+++ b/rich/console.py
@@ -948,6 +948,15 @@ class Console:
             # return False for Jupyter, which may have FORCE_COLOR set
             return False
 
+        # If TTY_COMPATIBLE env var is set to "1", we assume a terminal.
+        tty_compatible = self._environ.get("TTY_COMPATIBLE")
+        if tty_compatible == "1":
+            self._force_terminal = True
+            return True
+        if tty_compatible == "0":
+            self._force_terminal = False
+            return False
+
         # If FORCE_COLOR env var has any value at all, we assume a terminal.
         force_color = self._environ.get("FORCE_COLOR")
         if force_color is not None:
