diff --git a/rich/default_styles.py b/rich/default_styles.py
index 3a0ad83a..26ee311d 100644
--- a/rich/default_styles.py
+++ b/rich/default_styles.py
@@ -159,7 +159,6 @@ DEFAULT_STYLES: Dict[str, Style] = {
     "markdown.h4": Style(bold=True, dim=True),
     "markdown.h5": Style(underline=True),
     "markdown.h6": Style(italic=True),
-    "markdown.h7": Style(italic=True, dim=True),
     "markdown.link": Style(color="bright_blue"),
     "markdown.link_url": Style(color="blue", underline=True),
     "markdown.s": Style(strike=True),
